packages/design-system/src/connect/components/document-toolbar/utils/use-document-undo-redo.ts

```diff
@@ -0,0 +1,27 @@
+import { useDocumentById } from "@powerhousedao/reactor-browser";
+import { redo, undo } from "document-model/core";
+
+export function useDocumentUndoRedo(documentId?: string) {
+  const [document, dispatch] = useDocumentById(documentId);
+
+  const globalRevisionNumber = document?.header.revision.global ?? 0;
```

Comment:
I'm not sure how this will work out in the future as there might not even be a `global` scope. Let's keep it for now but we should add a test to detect regressions.